docs-source/content/faq/fan.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Disabling Fast Application Notifications"
+date:
+draft: false
+---
+
+To support Fast Application Notifications (FAN), Oracle databases configure GRID.
+GRID is typically associated with (and required by) Oracle RAC databases but can
+also be used in other configurations.  ATP-S does not provide GRID.
+
+When connecting to a database that does not have GRID, the only type of WebLogic Server
+supported data source is Generic Data Sources. Multi Data Sources and Active GridLink
+data sources cannot be used because they work with RAC.
+
+
+When connecting with a 12.2 driver to a database that does not have GRID, you will
```

Comment:
WebLogic Server 12.2.1.3.0 shipped with the 12.2.0.1 Oracle driver, when connecting with this driver ...